packages/ui-extensions/docs/surfaces/checkout/build-docs.sh

```diff
@@ -14,7 +14,10 @@ fail_and_exit() {
 run_sed() {
   if [[ "$OSTYPE" == "darwin"* ]]; then
     # macOS
-    sed -Ei '' "$1" "$2"
```

Comment:
I think we want to avoid this one right @kumar303 ? I think I commited this one by mistake

Reply:
haha the answer is in the comment, will remove